src/pipx/commands/deselect.py

```diff
@@ -0,0 +1,64 @@
+from pipx import constants
+from pipx.colors import red, bold
+from pipx.commands.common import find_selected_venvs_for_package, expose_package_globally, unexpose_package_globally
+from pipx.util import PipxError
+from pipx.venv import VenvContainer, Venv
+
+
+def deselect(
+    package_name: str,
```

Comment:
As per description in `main.py`: `package_name` can be a bare package name or have a suffix.